zebra-chain/src/tests/vectors.rs

```diff
@@ -30,7 +30,7 @@ impl Network {
         }
     }
 
-    ///
+    /// Returns BTreemap of blocks.
```

Comment:
###### Optional Suggestion

```suggestion
    /// Returns blocks indexed by height in a [`BtreeMap`].
    ///
    /// Returns Mainnet blocks if `self` is set to Mainnet, and Testnet blocks otherwise.
```